data/team.yml

```diff
@@ -33,106 +32,67 @@ teamLeaders:
         - icon : "tf-ion-social-github"
           URL : "https://github.com/lnaden"
 
+    - image: "images/people/lorifull-ish.jpg"
+#      imageAlt: "images/people/lorifull-ish.webp"
+      name: "Lori Burns"
+      designation: "Lead Psi4 Developer"
+      content: "Lead Psi4 dev and developer for the QC Engine and QC Elemental projects"
```

Comment:
"Developer for the QCEngine and QCElemental projects." No spaces or abbreviations.

Reply:
if you include Psi4, don't promote me beyond "a Psi4 core dev", please. and if you need a website anywhere, use my GH. it's the only thing that knows what I'm doing.